### PR TITLE
feat: trim keys

### LINKED ISSUE #22: Add trim to key combinations
I think we could add a `.map(str::trim)` here to make it more robust.

https://github.com/friendlymatthew/leptos-hotkeys/blob/e22eb0aed3055c434edaba4c8a66cbd80667aaa7/src/use_hotkeys.rs#L10

### PR DESCRIPTION
**Pull request by friendlymatthew**
Fixes #22

### CODE REVIEW

File: example/demo/src/app.rs [hunk 1]
```diff
@@ -71,13 +71,13 @@ fn HomePage() -> impl IntoView {
     );
 
     // scope_a related hotkeys
-    use_hotkeys!(("arrowup", "scope_a") => move |_| {
+    use_hotkeys!(("arrowup,arrowright", "scope_a") => move |_| {
```
Comment (friendlymatthew) [inline example/demo/src/app.rs:74]: btw @JustBobinAround we can bind multiple key combos with the `,` sym.

This means `ctrl+k,arrowup` means ctrl+k and uparrow both trigger the same callback

Comment (JustBobinAround) [reply example/demo/src/app.rs:74]: I thought this happens already:
```rust
pub fn use_hotkeys_ref_scoped<T>( 
    key_combination: String,
    on_triggered: Callback<()>,
    scopes: Vec<String>,
) -> NodeRef<T>
where
    T: ElementDescriptor + 'static + Clone,
{
    let node_ref = create_node_ref::<T>();

    create_effect(move |_| {
        let parsed_keys: HashSet<Hotkey> = key_combination
            .split(',') // <-- Right here and is the same for use_hotkeys_scoped
            .map(|key_combo| parse_key(key_combo))
            .collect();
```
Maybe I'm misunderstanding?

Comment (friendlymatthew) [reply example/demo/src/app.rs:74]: Nope! You're correct. Just wanted to state because none of the hotkeys in the demo do multiple

Comment (JustBobinAround) [reply example/demo/src/app.rs:74]: I think he just wants to trim whitespace, so just add a trim() after split

**Pull request merged**